src/config.js

```diff
@@ -23,6 +23,13 @@ module.exports.config = {
     },
 };
 
+/**
+ * @returns {object} The config object
+ */
+module.exports.getConfig = function getConfig() {
```

Comment:
this is ok, but make the config objects not be module.exports.config but just config